src/status_im2/contexts/wallet/add_address_to_watch/confirm_address/view.cljs

```diff
@@ -14,13 +13,14 @@
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
+(def ^:private no-password nil)
```

Comment:
What is `no-password` used for? I see it is initialized to `nil` and just passed to `rf/dispatch [:wallet/add-account`

Reply:
it's to explain the var, I'll either remove or move to a map 👍